proposals/71:upgrade-14/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "agoricProposal": {
+    "releaseNotes": "https://github.com/Agoric/agoric-sdk/releases/tag/agoric-upgrade-14",
+    "sdkImageTag": "41",
+    "planName": "agoric-upgrade-14",
+    "upgradeInfo": {},
+    "type": "Software Upgrade Proposal"
+  },
+  "type": "module",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@agoric/synthetic-chain": "^0.0.7",
+    "ava": "^5.3.1"
+  },
+  "ava": {
+    "concurrency": 1
```

Comment:
TMK it doesn't do anything. https://github.com/avajs/ava/blob/main/docs/06-configuration.md

Reply:
It says:
> All of the [CLI options](https://github.com/avajs/ava/blob/main/docs/05-command-line.md) can be configured in the `ava` section of either your `package.json` file, or an `ava.config.*` file.

While it's not listed in the example doesn't mean it doesn't do anything.

Reply:
I had read that but because most of other CLII options were included in the table and `serial` was not, I took it to mean that `serial` was an exception. It's also missing from https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/ava.json

I see in the code that it is observed so I'll PR on Ava and revert this removal.
https://github.com/avajs/ava/blob/01ec2804ab9db0ab3ef11e3b5b0c5697d68f8bc5/lib/cli.js#L438